src/main/java/su/litvak/chromecast/api/v2/ChromeCasts.java

```diff
@@ -35,9 +36,13 @@
     private ChromeCasts() {
     }
 
-    private void _startDiscovery() throws IOException {
+    private void _startDiscovery(InetAddress addr) throws IOException {
```

Comment:
Why have you changed this? I don't think it is related with missing fields anyhow. Can you please file it in a separate pull request?